src/components/DocumentationTopic/TopicsLinkCardGridItem.vue

```diff
@@ -0,0 +1,95 @@
+<!--
+  This source file is part of the Swift.org open source project
+
+  Copyright (c) 2022 Apple Inc. and the Swift project authors
+  Licensed under Apache License v2.0 with Runtime Library Exception
+
+  See https://swift.org/LICENSE.txt for license information
+  See https://swift.org/CONTRIBUTORS.txt for Swift project authors
+-->
+
+<template>
+  <Card
+    class="reference-card-grid-item"
+    :url="item.url"
+    :image="imageReferences.card"
```

Comment:
Found a potential issue trying to test this, but I'm not 100% sure if it's a DocC bug or something I'm doing wrong.

Using the latest swift-docc main, I'm testing out these changes:
```diff
diff --git a/Sources/DocCDocumentation/DocCDocumentation.docc/DocC Documentation.md b/Sources/DocCDocumentation/DocCDocumentation.docc/DocC Documentation.md
index 704ad66d..4de9c69f 100644
--- a/Sources/DocCDocumentation/DocCDocumentation.docc/DocC Documentation.md	
+++ b/Sources/DocCDocumentation/DocCDocumentation.docc/DocC Documentation.md	
@@ -2,6 +2,10 @@
 
 Produce rich API reference documentation and interactive tutorials for your Swift framework or package.
 
+@Options {
+  @TopicsVisualStyle(compactGrid)
+}
+
 ## Overview
 
 The DocC documentation compiler converts Markdown-based text into rich documentation for Swift frameworks and packages. You can preview the documentation in its published form as you work on it, and also host it on a website when it's complete.
diff --git a/Sources/DocCDocumentation/DocCDocumentation.docc/formatting-your-documentation-content.md b/Sources/DocCDocumentation/DocCDocumentation.docc/formatting-your-documentation-content.md
index 95e23c35..8ae4dc6c 100644
--- a/Sources/DocCDocumentation/DocCDocumentation.docc/formatting-your-documentation-content.md
+++ b/Sources/DocCDocumentation/DocCDocumentation.docc/formatting-your-documentation-content.md
@@ -2,6 +2,10 @@
 
 Enhance your content's presentation with special formatting and styling for text, links, and other page elements.
 
+@Metadata {
+  @PageImage(purpose: card, source: "building-tutorial", alt: "Alt text")
+}
+
 ## Overview
 
 Use [Markdown](https://daringfireball.net/projects/markdown/syntax), a 
```

And it doesn't look like the card image is working for the one where I added one:
<img width="1436" alt="Screenshot 2022-09-16 at 3 26 31 PM" src="https://user-images.githubusercontent.com/212918/190826263-e2ea97ab-239e-4ee4-b205-fe65027705e6.png">

@ethan-kusters @dobromir-hristov is there anything obviously wrong with the way I've authored this? It looks like both PRs have been merged on DocC for the options/metadata directives but the card image doesn't seem to be working quite right.

Reply:
That was my error @mportiz08, fixed in - 9a48324. Thanks for noticing it.

Reply:
Nice, looks like it was actually a very simple fix.